README.md

```diff
@@ -1,2 +1,3 @@
 # sinatra_app
 Sample sinatra application
+Heroku link: https://supersinatra.herokuapp.com/

```

Comment:
Ok. In future i will be give comments how to install and use application on the local pc. Now I will add description to my Project.